Makefile

```diff
@@ -98,7 +98,7 @@ lint:
 	docker run --rm -v $(shell pwd):/app \
 		-v ${GOLANGCI_LINT_CACHE}:/root/.cache/golangci-lint \
 		-w /app golangci/golangci-lint:${GOLANGCI_LINT_VERSION}-alpine \
-		golangci-lint run -v
```

Comment:
hmm, I think walter already raised made this exact change byt curious why GitHub is not displaying it 🤔 

https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/e7735cc9b7e35671a2aac7f862de23b67e95fcee/Makefile#L101

Anyhow, the better way for us to do this is probably modify this line instead:

https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/e7735cc9b7e35671a2aac7f862de23b67e95fcee/.golangci.yaml#L16

Reply:
Perfect! I'll abandon this change!

> but curious why GitHub is not displaying it

Probably because I'm not synced to HEAD locally and edited a different file.